packages/e2e/features/ui/components/authenticator/sign-in-force-new-password.feature

```diff
@@ -6,22 +6,35 @@ Feature: Sign In with Force New Password flow
 
   Background:
     Given I'm running the example "ui/components/authenticator/sign-in-with-phone"
+    Given I intercept '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.RespondToAuthChallenge" } }' with fixture "force-change-password"
 
-  @angular @react @vue @skip
+  @angular @react @vue 
+  Scenario: Back to Sign In works in the FORCE_CHANGE_PASSWORD state
+    When I select my country code with status "FORCE_CHANGE_PASSWORD"
+    And I type my "phone number" with status "FORCE_CHANGE_PASSWORD"
+    And I type my password
+    And I click the "Sign in" button
+    And I should see the Force Change Password screen
+    And I click the "Back to Sign In" button
+    And I see "Sign In"
+    
+
+  @angular @react @vue 
   Scenario: Sign in using a valid phone number and password and user is in a FORCE_CHANGE_PASSWORD state
     When I select my country code with status "FORCE_CHANGE_PASSWORD"
     And I type my "phone number" with status "FORCE_CHANGE_PASSWORD"
     And I type my password
     And I click the "Sign in" button
 		Then I should see the Force Change Password screen
 
-  @angular @react @vue @skip
+  @angular @react @vue 
   Scenario: User is in a FORCE_CHANGE_PASSWORD state and then enters an invalid new password
     When I select my country code with status "FORCE_CHANGE_PASSWORD"
     And I type my "phone number" with status "FORCE_CHANGE_PASSWORD"
     And I type my password
     And I click the "Sign in" button
+		Then I should see the Force Change Password screen
```

Comment:
I think there is some spacing problem here

Reply:
Will do!